src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -278,6 +277,16 @@
     <Compile Include="$(BclSourcesRoot)\System\ValueType.cs" />
     <Compile Include="$(BclSourcesRoot)\System\WeakReference.cs" />
     <Compile Include="$(BclSourcesRoot)\System\WeakReferenceOfT.cs" />
+    <Compile Include="shared\Interop\Windows\OleAut32\Interop.SysAllocStringByteLen.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.CoTaskMemAlloc.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.GetCurrentProcessId.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.GetCurrentThreadId.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.GetStdHandle.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.INVALID_HANDLE_VALUE.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.LocalAlloc.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.QueryUnbiasedInterruptTime.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.VerifyVersionExW.cs" />
+    <Compile Include="shared\Interop\Windows\Kernel32\Interop.VerSetConditionMask.cs" />
```

Comment:
I wasn't sure whether to put these in the shared project or here, since they're currently not used by other files in the shared corelib... I can move them if desired, though.

Reply:
I think this is fine.